source/_docs/scripts/conditions.markdown

```diff
@@ -400,9 +400,12 @@ This is 'when dark' - equivalent to a state condition on `sun.sun` of `below_hor
 
 ```yaml
 condition:
-  - condition: sun
-    after: sunset
-    before: sunrise
+  condition: or
+  conditions:
+    - condition: sun
+      after: sunset
+    - condition: sun
+      before: sunrise
```

Comment:
This should work and is a bug (which is already reported in the core repository.

Reply:
@frenck that part is currently saying it will always return false which I don't think is right:

![image](https://user-images.githubusercontent.com/567982/176288981-b52cfbfa-ce7a-4def-88b3-453722355c18.png)

Is it this bit that has been reported as a bug?
![image](https://user-images.githubusercontent.com/567982/176289379-96848394-41ef-4021-abfc-a34833743cbb.png)

Reply:
@dannytsang I'd say best to remove the "We cannot use both keys..." and just replace it with the original "This is 'when dark'..." text?

Reply:
@smartroad removed the example in the latest changes. Sorry @frenck I merged a bunch of commits when I probably should have rebase the changes first.